t/data/tests/tests/shutdown.pm

```diff
@@ -18,6 +18,7 @@ use base 'basetest';
 use testapi;
 
 sub run {
+    wait_idle 1;
```

Comment:
IMHO a waste of time. wait_idle should be tested in t/03-testapi.t with mocked sleep

Reply:
So you don't want to comment on this?